### PR TITLE
feat: added pinning service request issue mailto link/github button

### PR DESCRIPTION
**Pull request by LeslieOA**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -3,3 +3,6 @@ contact_links:
   - name: Account Limit Increase Request
     url: https://mailxto.com/xy5ijb
     about: Submit a request to increase the size limit on your Web3.Storage account!
+  - name: Pinning Service Access Request
+    url: https://mailxto.com/h58kmo
+    about: Submit a request to access Web3.Storage's IPFS pinning service in your account!
```
Comment (alexandrastoica) [inline .github/ISSUE_TEMPLATE/config.yml:8]: needs a new line?

**Review by GaryHomewood — APPROVED** [review]
All good.

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (LeslieOA) [reply .github/ISSUE_TEMPLATE/config.yml:8]: @alexandrastoica Functionally, it doesn't; unless you mean file linting? 😬

Comment (alexandrastoica) [reply .github/ISSUE_TEMPLATE/config.yml:8]: Just linting, saw it flagged in the diff

**Pull request merged**